earth_enterprise/src/fusion/rasterfuse/TmeshPackgenTraverser.h

```diff
@@ -75,13 +75,18 @@ class TmeshWorkItem {
   khDeletingVector<Piece> pieces;
   uint numPiecesUsed;
   TmeshGenerator generator;
+  bool decimate_;
+  float decimation_threshold_;
 
   TmeshWorkItem(PacketFileWriter &_writer,
                 const AttributionByExtents &_attributions,
-                const khTilespaceFlat &_sampleTilespace, bool decimate);
+                const khTilespaceFlat &_sampleTilespace,
+                bool decimate,
+                float decimation_threshold);
```

Comment:
can decimation_threshold change per TmeshWorkItem ?  Its already part of constructor  TmeshPackgenTraverser , so if it need to remain same thruout the class, we dont need it here, which is my understanding.

Reply:
How else do you propose passing it to Generate() if not through the class that calls Generate()?  TmeshPackgenTraverser is the middleman between the command line and this point in the code.